airbyte-json-validation/src/main/java/io/airbyte/validation/json/JsonSchemaValidator.java

```diff
@@ -26,12 +31,24 @@ public class JsonSchemaValidator {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(JsonSchemaValidator.class);
 
-  private final SchemaValidatorsConfig schemaValidatorsConfig;
   private final JsonSchemaFactory jsonSchemaFactory;
+  private final URI BASE_URI;
 
   public JsonSchemaValidator() {
-    this.schemaValidatorsConfig = new SchemaValidatorsConfig();
+    // This URI just needs to point at any path in the same directory as /app/WellKnownTypes.json
+    // It's required for the JsonSchema#validate method to resolve $ref correctly.
+    this("file:///app/nonexistent_file.json");
+  }
+
+  @VisibleForTesting
+  protected JsonSchemaValidator(String baseUri) {
```

Comment:
Passing an `URI` here instead of a String would be a clearer contract and it removes the need for the exception wrapping in the constructor.

Reply:
done - there's a static initializer for the default uri but I think that's fine since it's hardcoded to a valid value